README.md

```diff
@@ -7,19 +18,15 @@ environment.
 
 This repository contains tools to create a guest OS disk image.
 
-## Table of Contents
-* [Terms](#terms)
-
 ## Terms
 
 This section describes the terms used for all documentation in this repository.
 
 - rootfs
 
-  The root filesystem or "rootfs" is the set of files contained in the
-  guest root directory that builds into a filesystem.
+  The root filesystem or "rootfs" is a slight misnomer as it is not a true filesystem: it is simply a tree of files contained in a particular directory which represents the root disk layout. It can be turned into either an image or an initrd.
```

Comment:
Suggested rewrite: The root filesystem or "rootfs" is a slight misnomer as it is not a true filesystem. It is a tree of files contained in a particular directory, which represents the root disk layout. Rootfs can turn into either an image or an initrd.